Userland/wc.cpp

```diff
@@ -85,6 +85,10 @@ static Count get_count(const String& file_specifier)
             count.words++;
         }
     }
+
+    if (file_pointer != stdin)
+        fclose(file_pointer);
```

Comment:
I'd maybe call this a file leak, not just a memory leak :)
But it's not terribly important.

Reply:
This is both a file descriptor leak and a memory leak I think. Because `file_pointer` is allocated in the heap.

I found this using `UserspaceEmulator` that called it a memory leak so I just used that term.